lepton/image.go

```diff
@@ -525,3 +556,19 @@ func lookupFile(targetRoot string, path string) (string, error) {
 
 	return path, err
 }
+
+// saveImageConfig saves image config as JSON
+// for volume attach/detach purposes
+func saveImageConfig(c Config) error {
```

Comment:
this is fine for now but something we should prob. fix in the future - a good chunk of the variables being stored here are dynamic in nature and don't belong here;

also - 'manifests' have an inherent meaning already and probably wise to change this to a different phrase